wo8uy7-d1e549/cod-LyXLXy_wo8uy7-d1e549.xml

```diff
@@ -1,590 +1,775 @@
+          <lb ed="#L" n="32"/>ponenda est ymago. sed in potentiis est maior
+          con<lb ed="#L" n="33" break="no"/>substantialitas. quia siue sint accidentia siue non.
+          imme<lb ed="#L" n="34" break="no"/>diatius se habent ad essentiam anime. Est etiam maior
+          per<lb ed="#L" n="35" break="no"/>petuitas. quia semper manent. non sicut actus nec
+          ha<lb ed="#L" n="36" break="no"/>bitus quicumque. Similiter est in eis maior conformitas
+          <lb ed="#L" n="37"/>quia est ibi distinctio et ordo. sine productione de
+          po<lb ed="#L" n="38" break="no"/>tentia ad actum.
+        </p>
+       
+       
+       
+        <p xml:id="wo8uy7-d1e549-d1e259">
+          <note place="margin-left">Instatur primo</note>
+          ¶ Sed ista opinio non videtur esse
+          <lb ed="#L" n="39"/>vera. quia non sunt in anima tales tres potentie. quia sicut
+          <lb ed="#L" n="40"/>declarabitur in secundo nulla est distinctio praeuia in
+          <lb ed="#L" n="41"/>ipsa substantia anime. ante distinctionem actuum
+          secun<lb ed="#L" n="42" break="no"/>dorum productorum. ergo non sunt ibi talia tria qua
+          <lb ed="#L" n="43"/>repraesentant diuinas personas.
+        </p>
+       
+       
+       
+        <p xml:id="wo8uy7-d1e549-d1e272">
+          <note place="margin-left">Secundo</note>
+          ¶ Preterea. quando aliquid
+          <lb ed="#L" n="44"/>ponitur ymago alicuius propter distinctionem i <!--this is not "in" but "primam"-->
+          yma<lb ed="#L" n="45" break="no"/>gine corresponden ter distinctioni in illo cuius est yma
+          <lb ed="#L" n="46"/>go tanta debet esse distinctio inter partes ymagis
+          quanta<lb ed="#L" n="47" break="no"/>i illo cus ymago. sed inter personas diuinas est distinctio
+          <lb ed="#L" n="48"/>realis. non inter potentias anime. ergo propter distinctionem
+          po<lb ed="#L" n="49" break="no"/>tentiarum non est ponenda in anima ymago.
+        </p>
+       
+       
+       
+        <p xml:id="wo8uy7-d1e549-d1e288">
+          <note place="margin-left">D</note>
+          <note place="margin-left">Respondetur ad quaestionem per duo</note>
+          ¶ Ideo
+          <lb ed="#L" n="50"/>dico aliter ad questionem. Primo enim quomodo anima vel
+          <lb ed="#L" n="51"/>creatura rationalis plus dicitur ymago dei quam aliqua
+          <lb ed="#L" n="52"/>alia creatura. Secundo ratione cuius vel quorum est
+          yma<lb ed="#L" n="53" break="no"/>go.
+        </p>
+       
+       
+       
+        <p xml:id="wo8uy7-d1e549-d1e300">
+          <note place="margin-left">Primum</note>
+          ¶ Circa primum dico quod cum pateat ex predictis
+          <lb ed="#L" n="54"/>quod ymago secundum quod inuenitur in creaturis dicitur propter
+          <lb ed="#L" n="55"/>conformitatem alicuius eiusdem rationis in illo cuius est
+          <!--276.xml-->
```

Comment:
eventually we will delete this, but we can leave them for a while longer as they help me find the page number, when the folio number system is not used.